ui/.eslintrc.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
```

Comment:
a jakbyśmy chcieli korzystać z nowszego es niż 6? to ten tutaj się ustawia?

Reply:
This is just an eslint configuration. `tsconfig.json` is responsible for the Typescript itself. Also below `ecmaVersion` is set to 2018.